CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ if(WIN32)
 else()
   cmake_minimum_required(VERSION 3.1)
```

Comment:
On the current `develop` branch, we're now requiring CMake 3.9.  Would be good to update it here as well so we do not accidentally revert when we merge later.

Reply:
Thanks. I will update this.